applications/mne_fiff_anonymizer/mne_fiff_anonymizer.pro

```diff
@@ -0,0 +1,78 @@
+#--------------------------------------------------------------------------------------------------------------
+#
+# @file     mne_fiff_anonymizer.pro
+# @author   Juan Garcia-Prieto <JuanGarciaPrieto@uth.tmc.edu>;
+#           Matti Hamalainen <msh@nmr.mgh.harvard.edu>
+# @version  1.0
+# @date     August, 2019
+#
+# @section  LICENSE
+#
+# Copyright (C) 2019, Juan Garcia-Prieto and Matti Hamalainen. All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without modification, are permitted provided that
+# the following conditions are met:
+#     * Redistributions of source code must retain the above copyright notice, this list of conditions and the
+#       following disclaimer.
+#     * Redistributions in binary form must reproduce the above copyright notice, this list of conditions and
+#       the following disclaimer in the documentation and/or other materials provided with the distribution.
+#     * Neither the name of MNE-CPP authors nor the names of its contributors may be used
+#       to endorse or promote products derived from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED
+# WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A
+# PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT,
+# INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+# HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+# NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+#
+#
+# @brief    This project file generates the makefile to build the mne_fiff_anonymizer example.
+#
+#--------------------------------------------------------------------------------------------------------------
+
+include(../../mne-cpp.pri)
+
+TEMPLATE = app
+
+QT += widgets 3dextras
```

Comment:
Since you included your application in the mininmalVersion of mne-cpp here, `3dextras` from the Qt3D module is not available. Also  we do not `3dextras` because do not link against the disp3d library. So you can safely remove `3extras` here. Also the minimalVersion should only include non GUI related tools and libraries. Consider removing `widgets` as well or move the application to the non minimalVersion in applications.pro file.

Reply:
I didn't know I had included it in the minimalVersion, however if it is there I think it might come in handy to someone (hope). It is only command-line. @imsorryk was going to remove 3dextras and widgets. Thanks for the comment. (I'll ask you sometime later how did I include the app in the minimal version).

Reply:
Alright. Let's keep it in the minimalVersion for now. If you want to create a small GUI mode in the future we can always bump it up to the full version.